Sage/carbon
# fix(grid-container): children proptype definition (FE-3222)

fixes: #3272

### Proposed behaviour
This fix clears the console warning when a single GridItem is passed to the children prop by correctly defining the children proptype as GridItem or Array of GridItem. 

### Current behaviour
Single GridItem as a child of GridContainer throws this console warning
`Warning: Failed prop type: Invalid prop `children` supplied to `GridContainer`.`

### Checklist


~- [ ] Screenshots are included in the PR if useful~
~- [ ] Cypress automation tests added or updated if required~
~- [ ] Storybook added or updated if required~
~- [ ] Typescript `d.ts` file added or updated if required~

### Additional context


### Testing instructions
Use the codesandbox generated for issue 3272 below, edit it to pass either a single GridItem or multiple GridItems there should no longer be a console warning.
